--- a/view/frontend/web/lib/owl.carousel.js	Fri Sep 08 12:51:03 2017 +0300
+++ b/view/frontend/web/lib/owl.carousel.js	Fri Sep 08 13:31:36 2017 +0300
@@ -1900,7 +1900,7 @@
 
 		$elements.each($.proxy(function(index, element) {
 			var $element = $(element), image,
-				url = (window.devicePixelRatio > 1 && $element.attr('data-src-retina')) || $element.attr('data-src');
+                url = (window.devicePixelRatio > 1 && $element.attr('data-src-retina')) || $element.attr('data-src') || $element.attr('data-srcset');
 
 			this._core.trigger('load', { element: $element, url: url }, 'lazy');
 
@@ -1909,7 +1909,11 @@
 					$element.css('opacity', 1);
 					this._core.trigger('loaded', { element: $element, url: url }, 'lazy');
 				}, this)).attr('src', url);
-			} else {
+			} else if($element.is('source')) {
+                $element.one('load.owl.lazy', $.proxy(function() {
+                    this._core.trigger('loaded', { element: $element, url: url }, 'lazy');
+                }, this)).attr('srcset', url);
+            } else {
 				image = new Image();
 				image.onload = $.proxy(function() {
 					$element.css({
